nilearn/datasets/atlas.py

```diff
@@ -773,3 +773,136 @@ def fetch_coords_dosenbach_2010():
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_coords_gordon_2014():
+    """Load the Gordon et al 2014 ROI centroids.
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, contains:
+        - "ids": the ID of the ROIs in the original atlas
+        - "rois": coordinates of 160 ROIs in MNI space
+        - "networks": networks names
+        - "hemisphere": L or R (left or right)
+        - "surface": the surface of the ROIs in millimeters
+
+    See also
+    --------
+        nilearn.datasets.fetch_atlas_gordon_2014
+
+    References
+    ----------
+    Gordon, E. M., Laumann, T. O., Adeyemo, B., Huckins, J. F., Kelley, W. M., &
+    Petersen, S. E., "Generation and evaluation of a cortical area
+    parcellation from resting-state correlations", 2014, Cerebral cortex, bhu239.
+    """
+    dataset_name = 'gordon_2014'
+    fdescr = _get_dataset_descr(dataset_name)
+    package_directory = os.path.dirname(os.path.abspath(__file__))
+    csv = os.path.join(package_directory, "data", "gordon_2014.csv")
```

Comment:
May be we should add details about the data in csv file. From where is it retrieved, etc ?